dpnp/dpnp_iface.py

```diff
@@ -69,6 +69,7 @@
     "get_usm_ndarray_or_scalar",
     "is_supported_array_or_scalar",
     "is_supported_array_type",
+    "_copyto",
```

Comment:
I would suggest to rename, like:
```suggestion
    "get_result_array",
```
since it is not always about the copying to `out`